src/main/java/net/consensys/orion/http/handler/send/SendHandler.java

```diff
@@ -14,268 +14,38 @@
 
 import static net.consensys.orion.http.server.HttpContentType.JSON;
 
-import net.consensys.cava.crypto.sodium.Box;
-import net.consensys.orion.config.Config;
-import net.consensys.orion.enclave.Enclave;
-import net.consensys.orion.enclave.EncryptedPayload;
-import net.consensys.orion.enclave.PrivacyGroupPayload;
-import net.consensys.orion.enclave.QueryPrivacyGroupPayload;
-import net.consensys.orion.exception.OrionErrorCode;
-import net.consensys.orion.exception.OrionException;
-import net.consensys.orion.http.server.HttpContentType;
-import net.consensys.orion.network.ConcurrentNetworkNodes;
-import net.consensys.orion.network.NodeHttpClientBuilder;
-import net.consensys.orion.storage.Storage;
 import net.consensys.orion.utils.Serializer;
 
-import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-import java.util.concurrent.CompletableFuture;
-import java.util.stream.Collectors;
-
 import io.vertx.core.Handler;
-import io.vertx.core.Vertx;
-import io.vertx.core.buffer.Buffer;
-import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.Json;
 import io.vertx.ext.web.RoutingContext;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 
-/** Send a base64 encoded payload to encrypt. */
+/**
+ * Send a base64 encoded payload to encrypt.
+ */
```

Comment:
Remove this? Or at least make it accurate - eg "handle a received /Send request, and forward payload to recipients"

Reply:
Probably remove. The code for this class is simple enough now that anyone can read it and understand what it is doing (or at least I hope it is) :P